specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,835 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataPolicy_Delete": {
+            "$ref": "./examples/DataPolicy_Delete.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "DataPolicy"
+        ],
+        "description": "Gets a data policy",
+        "operationId": "DataPolicy_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "policyName",
+            "description": "Id of an existing policy that needs to be fetched.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicy"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataPolicy_Get": {
+            "$ref": "./examples/DataPolicy_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AttributeMatcher": {
+      "description": "Attribute matcher for a rule",
+      "type": "object",
+      "properties": {
+        "attributeName": {
+          "description": "AttributeName",
+          "type": "string"
+        },
+        "attributeValueIncludes": {
+          "description": "Value for attribute",
+          "type": "string"
+        },
+        "attributeValueIncludedIn": {
+          "description": "List of values for attribute",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "attributeValueExcludes": {
+          "description": "Value excluded for attribute",
+          "type": "string"
+        },
+        "attributeValueExcludedIn": {
+          "description": "List of values excluded for attribute",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "DnfCondition": {
```

Comment:
Are those abbreviations the dominant term in this technology domain and known by your customers? Consider spelling out or renaming. Same for CNF below.

Reply:
Our customers have already been using these terms for some time now.
Here is a spec that we pushed last year.

https://github.com/Azure/azure-rest-api-specs/blob/24b1f9ed9a0768a55b14cced1761a40f7c12b6d2/specification/purview/data-plane/Azure.Analytics.Purview.MetadataPolicies/preview/2021-07-01-preview/purviewMetadataPolicy.json#L247